### PR TITLE
Update with chart

### PR DESCRIPTION
**Pull request by atoy3731**

### CODE REVIEW

File: chart/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: releasebot
+description: Helm chart for the ReleaseBot
+type: application
+version: 0.1.0
+appVersion: "1.16.0"
```
Comment (clanktron) [inline chart/Chart.yaml]: probably set appversion to 0.1.0

File: cmd/github.go [hunk 2]
```diff
@@ -173,14 +173,16 @@ func monitor_repo(owner string, repo string, prereleases bool, tekton bool) {
...
+		fmt.Printf("Found a new release for %s/%s (%s)\n", owner, repo, newrelease.Name)
+		if slack {
+			slacknotif(newrelease, owner, repo, prereleases_channel)
+		}
 		if tekton {
 			triggertekton(newrelease, owner, repo)
 		}
 	} else {
-		fmt.Printf("No new prereleases for %s/%s\n", owner, repo)
+		fmt.Printf("No new releases for %s/%s\n", owner, repo)
```
Comment (clanktron) [inline cmd/github.go:186]: ill add a conditional to print either "release" or "prerelease"

File: dockerfile [hunk 3]
```diff
@@ -1,11 +1,20 @@
...
 
-FROM alpine:latest
+COPY . /build
+
+WORKDIR /build
+
+ENV GOOS=linux
+ENV CGO_ENABLED=0
+
+RUN go get -d -v ./...
```
Comment (clanktron) [inline dockerfile:10]: I prefer to keep this stuff behind makefile targets so I only have to change one file. Those flags and the go get are good additions tho

File: chart/Chart.yaml (hunk 1 above)
Comment (atoy3731) [reply chart/Chart.yaml]: Fixed.

File: dockerfile (hunk 3 above)
Comment (atoy3731) [reply dockerfile:10]: Rev'd back to using `make`, but it _did_ also requite a `git` prereq. Not a big deal since we're doing multi-stage, but weird.

**Pull request merged**